test/new-e2e/tests/installer/script/all_scripts_test.go

```diff
@@ -91,6 +94,9 @@ func TestScripts(t *testing.T) {
 			t.Run(suite.Name(), func(t *testing.T) {
 				t.Parallel()
 
+				// INCIDENT(35594): To avoid rate limits, we need to wait between stack creation (aws imds rate limits)
+				time.Sleep(time.Duration(iteration) * stackCreationDelay)
```

Comment:
This probably won't work, the test is parallel so you're just going to delay all the test by 1min but they will happen at the same time